getblocklab/block-lab
# Add a migration notice on certain pages

#### Changes
* Adds the migration and 'Not Now' notices
* On clicking 'Not Now', the migration notice never appears again for that user:
* Appears on Block Lab > Content Blocks, Block Lab > Settings, `/wp-admin/plugins.php`, and the Dashboard. 






#### Testing instructions
1. Go to Block Lab > Content Blocks, Block Lab > Settings, `/wp-admin/plugins.php`, and the Dashboard
2. Expected: the notice appears as shown above
3. Click 'Not Now'
4. Expected: the notice is hidden, and the 'Not Now' notice displays
5. Reload the page
6. Expected: The notice doesn't appear again
7. To make the notice appear again:
```sh
wp user list # Find your user ID from that
wp shell
update_user_meta( $user_id, 'block_lab_show_migration_notice', 1 ); // $user_id is the ID from the wp user list command.
```
